content/en/docs/contribute/_index.md

```diff
@@ -4,59 +4,76 @@ title: Contribute to Kubernetes docs
+roles and permissions.
 
-- Do write clear and meaningful GIT commit messages.
-- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
-- When you make a small change to a PR like fixing a typo, any style change, or changing grammar. Make sure you squash your commits so that you dont get a large number of commits for a relatively small change.
-- Make sure you include a nice PR description depicting the code you have changes, why to change a following piece of code and ensuring there is sufficient information for the reviewer to understand your PR.
-- Additional Readings : 
-    - [chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.com/blog/1506-closing-issues-via-pull-requests )
-    - [davidwalsh.name/squash-commits-git ](https://davidwalsh.name/squash-commits-git )
+## Your first contribution
 
-## Other ways to contribute
+- [Suggest improvements](/docs/contribute/suggest-improvements/) to existing documentation and learn more about filing issues in GitHub.
+- Contribute a [small change](/docs/contribute/new-content/new-content/#small-changes) to existing content to learn the pull review process.
+- [Review pull requests](/docs/contribute/review/reviewing/) from other Kubernetes community members for accuracy and language.
+- Read the Kubernetes [content](/docs/contribute/style/content-guide/) and [style guides](/docs/contribute/style/style-guide/) so you can leave informed comments.
+- Learn how to [use page templates](/docs/contribute/style/page-templates/) and [Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/) to make bigger changes.
 
-- To contribute to the Kubernetes community through online forums like Twitter or Stack Overflow, or learn about local meetups and Kubernetes events, visit the [Kubernetes community site](/community/).
-- To contribute to feature development, read the [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) to get started.
+## Next steps
 
-{{% /capture %}}
+- Learn to [work from a local clone](/docs/contribute/new-content/working-locally/) of the repository.
+- Contribute a [larger change](/docs/contribute/new-content/new-content/#fork-the-repo).
+- Document [features in a release](/docs/contribute/new-content/new-features/).
+- Participate in [SIG Docs](/docs/contribute/participating/), and become a [member or reviewer](/docs/contribute/participating/#roles-and-responsibilities).
+- Start or help with a [localization](/docs/contribute/localization/).
+
+
+## Get involved with SIG Docs
+
+[SIG Docs](/docs/contribute/participating/) is the group of contributors who publish and maintain documentation. Getting involved with SIG Docs is a great way for Kubernetes contributors (feature development or otherwise) to have a large impact on the Kubernetes project.
```

Comment:
nit: Possible edit: Line 63, `who publish and maintain the Kubernetes documentation and website`.